test/src/auxkernels/StatefuleAux.C

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
```

Comment:
Uh oh, this file should be named `StatefulAux.C`

Reply:
fixed